docs/topics/proc-getting-started-update-booster.adoc

```diff
@@ -19,7 +19,7 @@ $ git clone git@github.com:USERNAME/PROJECT-NAME.git
 . Open your project in your desired IDE or editor, such as xref:use_devstudio[JBDS].
 . Make an update to the project.
 . Save your changes.
-. Commit and push your changes back to GitHub:
+. Commit and push your changes back to GitHub.
```

Comment:
Chicago Manual of Style says that sub-lists, examples, admonitions, and the like should be ushered by a colon, but I won't hold this PR on it.